Remotion/Web/Core/Res/Themes/NovaViso/HTML/TreeView.css

```diff
@@ -174,8 +174,8 @@ ul.treeViewRoot span.treeViewNode span.treeViewNodeBadge
   align-self: start;
 }
 
-ul.treeViewRoot li:has(a:focus) > ul > li:first-of-type span.treeViewNode:not(:hover) span.treeViewNodeBadge,
-ul.treeViewRoot li:has(a:focus) + li span.treeViewNode:not(:hover) span.treeViewNodeBadge
+ul.treeViewRoot li:has(> span.treeViewNode span[class^="treeViewNodeHead"] a[role=treeitem]:focus) > ul > li:first-of-type > span.treeViewNode:not(:hover) span.treeViewNodeBadge,
+ul.treeViewRoot li:has(> span.treeViewNode span[class^="treeViewNodeHead"] a[role=treeitem]:focus) + li:first-of-type > span.treeViewNode:not(:hover) span.treeViewNodeBadge
```

Comment:
I think the match for role=treeitem is overqualiciation.

Reply:
I think we should keep this, with the above changes I requested, we still need it, and it's probably cleaner this way

Reply:
hmm..yes, there's no short way to do this.

I looked into the perf-stats and now I'm worried about using `aria-selected` instead of treenodeheadselected css class. Maybe a ticket for this to be checked later?
![image](https://github.com/re-motion/Framework/assets/6972548/47432362-34df-4ac2-a5eb-8aec7d6b62e5)